### DISCUSSION TITLE
Linux: No way to copy-paste link URLs

### DISCUSSION THREAD

**Discussion by levu42** [spatie/ray#665] [Ideas]
When I display a Mailable in Laravel, I get a beautiful rendering of the HTML, however there are buttons in there that have a link address and the ray app (on linux) gives me no way of copy-pasting the URL, since there is no context menu. Copy pasting of other text works with Ctrl+C after marking it with the mouse, but it would be great to have at least a minimal context menu for copying the URL of a link.

**Versions**
Ray version (you can see this in "About Ray"): 1.19.0

You can use `composer show` to get the version numbers of:
- spatie/ray package version: 1.33.2
- spatie/laravel-ray package version (if applicable): 1.29.3

PHP version: 8.1.3
Laravel version (if applicable): 9.1

**To Reproduce**
1. ray()->mailable(new MailableWithSomeClickableButton())
2. Right click on the clickable button in the mail
3. See that there is no context menu

**Expected behavior**
Being able to right-click on something that's a link and select "Copy link destination" or something similar in a context menu

**Desktop (please complete the following information):**
 - OS: Linux (Arch)

**Comment by freekmurze** [comment]
Nice idea, we'll consider adding this.

**Comment by rico** [comment]
I would love to see this implemented as well. I always have to head over to the Herd mails to copy the links.
